### ISSUE TITLE
is it possible to add my user?

### ISSUE THREAD

**Issue by kurokirasama** [sudipghimire533/ytui-music#7]
so i can listen to my playlists?

**Comment by sudipghimire533** [comment]
Yeah. if your playlist is public. Copy the playlist id search it and add it to favourates. You are then good to go. Updating playlist from youtube is synced with ytui

**Comment by kurokirasama** [comment]
ok. What about my liked songs playlist or my super mix?

**Comment by sudipghimire533** [comment]
Depends on privacy set on your liked songs playlist. And what is super mix?

**Comment by sudipghimire533** [comment]
Thank you  @JPTIZ  for clarifying things.

About treating ytui as another front-end for youtube, the main thing of concern would be the API limit youtube had set. And I think we should stay away from it as this isn't the main intention of ytui.

About, accessing own data, Just like you I also miss my history and stuff. Doing one-time data importing might be go to approach if tolerated with possible lag with huge data.

Until now, closest one we can do is, making our playlist public in YouTube and searching the url and add that playlist to favourites.

Let's hear from @kurokirasama what he meant exactly.

**Comment by kurokirasama** [comment]
Hello, sorry the delay. What @JPTIZ  is correct, but accessing my own playlist would be enough for me. I tried with the public url but maybe I did something wrong. I'll try again.

**Comment by sudipghimire533** [comment]
See, There is my playlist https://www.youtube.com/playlist?list=PL61kECgFCR7fd-A0sIG9YYyAID0AuCyo-
And here is how I access it in ytui.

![image](https://user-images.githubusercontent.com/39845901/160978081-0031752c-c90c-4af5-977a-bb7164fc1c5b.png)

**Comment by kurokirasama** [comment]
Cool! I was putting the whole url there. Do you know any way to put all my liked YouTube music songs in a playlist?

**Comment by sudipghimire533** [comment]
Sorry but I am not aware of such

**Comment by sudipghimire533** [comment]
![image](https://user-images.githubusercontent.com/39845901/161087015-69db8262-8857-4d54-8dd6-17677919c74e.png)

This playlist seems to have unavailable videos only. Are you sure you are getting vidoes from something like guest window? Some videos might be unavailable in some country.

**Comment by kurokirasama** [comment]
I don't know... I just added them for youtube music app... I'll try again from youtube

**Comment by airtonix** [comment]
> If I'm not mixing names here (no pun intended), "Super mix" is an youtube feature that randomly creates a never-ending playlist (called "Mix") according to what it thinks you would like.

this is actually just called "Radio" in youtub music.

**Comment by erlonpr** [comment]
It would be interesting if there was an API integration with the Google account so that we could listen to our favorite playlists and songs.

Similar to what happens with Spotify TUI:
https://github.com/Rigellute/spotify-tui#connecting-to-spotifys-api

**Comment by dmr138** [comment]
Sigma67 has a ytmusic-api available that seems like it could help with this

**Comment by ArjixWasTaken** [comment]
> > If I'm not mixing names here (no pun intended), "Super mix" is an youtube feature that randomly creates a never-ending playlist (called "Mix") according to what it thinks you would like.
> 
> this is actually just called "Radio" in youtub music.

nope, radio is a different thing, it is similar but not the same
